### ISSUE TITLE
Magnifiers are not keyboard accessible

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1385]
Magnifiers on the following pages can not be controlled using keyboard only:

- Advanced image
- Graphics & sound
- Image sequence

**Comment by FayCross** [comment]
For info, these issues are noted on the 3.14 accessibility statement:

The Graphics & Sound, Advanced Image and Image Sequence pages may contain images that when hovered display a tooltip containing a magnified area of the image. The magnifier is not accessible to sighted keyboard-only users as it can only be triggered by the mouse movement. This fails [2.1.1 (Keyboard)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=222%2C211#keyboard-accessible). 

The image magnifiers cannot be dismissed without moving the mouse focus. This fails [1.4.13 (Content on Hover or Focus)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=144%2C1410%2C211%2C222%2C247#content-on-hover-or-focus).
